core/.js/src/main/scala/io/udash/bindings/inputs/Input.scala

```diff
@@ -7,26 +7,39 @@ import scalatags.JsDom.all._
 
 import scala.concurrent.duration.{Duration, DurationInt}
 
-/** Abstraction for HTML input tags.*/
+/** Abstraction for HTML input tags. */
 private[bindings] abstract class Input(inputType: String) {
   /**
-    * @param value Property to bind.
-    * @param debounce Property update timeout after input changes.
-    * @param inputModifiers Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
-    * @return HTML input with bound Property, applied modifiers and nested options.
-    */
-  def apply(value: Property[String], debounce: Duration = 20 millis)(inputModifiers: Modifier*): InputBinding[JSInput] =
+   * @param value               Property to bind.
+   * @param debounce            Property update timeout after input changes.
+   * @param onInputElementEvent Callback that's executed when `Input` element receives one of following events:
+   *                            `Input`, `Change`, `KeyUp`, `Paste` and element value is different than property value.
+   *                            Can be used to mimic unidirectional data flow for components based on this `Input`.
+   * @param inputModifiers      Additional modifiers. Don't use modifiers on value, onchange and onkeyup attributes
+   *                            as they are used internally to sync property value with element value.
```

Comment:
This part of docs is unclear, I'd rather have something like "if you pass modifiers on ..., then ...". They'd probably just get ignored, wouldn't they? (also maybe they don't have to and we can remove the comment then)

Also, the number of modifier types is different than the number of event types above, so you missed sth.